exporter/jaeger/jaegergrpcexporter/exporter.go

```diff
@@ -73,8 +76,14 @@ func (s *protoGRPCSender) pushTraceData(
 		return len(td.Spans), consumererror.Permanent(err)
 	}
 
+	octx := context.Background()
+
+	if len(s.headers) > 0 {
+		octx = metadata.NewOutgoingContext(octx, metadata.New(s.headers))
```

Comment:
nit: the return of `metadata.New` could be cached at creation instead of creating it on every push.

Reply:
Modified to create metadata once.